### PR TITLE
Prizes updates

### PR DESCRIPTION
**Pull request by caleybrock**
announce “prizes for every educator” 
- on code.org/learn
- on code.org/educate- only remove button
- hoc.com homepage next to signup - replace “What’s next?” with “Prizes” and maybe include logos/pictures

@tanyaparker can you check this and add the teacher announcement to it?

### CODE REVIEW

**Comment by caleybrock** [comment]
Still unsure of what prizes we are announcing.

File: pegasus/sites.v3/code.org/public/educate/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ title: Educator Overview
 
 The first step to introduce children to computer science is to participate in the Hour of Code, the largest learning campaign in history. In one hour, students (and teachers) can learn that computer science is fun, easy, and accessible at all ages.
 
-[<button>Learn more about Hour of Code</button>](http://hourofcode.com) &nbsp;&nbsp; [<button>How-to guide for educators</button>](http://hourofcode.com/resources/how-to)
+[<button>Learn more about Hour of Code</button>](http://hourofcode.com) &nbsp;&nbsp;
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/educate/index.md]: Can remove the `&nbsp;` spaces too

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css [hunk 2]
```diff
@@ -101,3 +101,17 @@
...
+}
+
+.apple {
+  width: 100px;
+}
+
+.amazon {
+  width: 190px;
+  padding-top: 16px;
+}
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: needs EOF new line

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -179,6 +179,9 @@
   prizes_nav_terms: "Terms and conditions"
   prizes_nav_faq: "Frequently asked questions"
 
+  prizes_header: "A prize for every educator"
+  prizes_description: "Giftcards provided by our partners at Amazon and Apple."
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: I think gift cards should be two words

**Pull request closed without merging**